packages/plugin-ext/src/common/plugin-api-rpc.ts

```diff
@@ -71,6 +72,9 @@ import { DebugProtocol } from 'vscode-debugprotocol';
 import { SymbolInformation } from 'vscode-languageserver-types';
 import { ScmCommand } from '@theia/scm/lib/browser/scm-provider';
 import { ArgumentProcessor } from '../plugin/command-registry';
+import { MaybePromise } from '@theia/core/lib/common/types';
+import { QuickOpenItem, QuickOpenItemOptions } from '@theia/core/lib/browser/quick-open/quick-open-model';
```

Comment:
we cannot reference browser from common

Sorry for bothering with it, but it is important. I've opened: https://github.com/theia-ide/theia/issues/5873 Maybe we can auto check it.

Reply:
fixed